app/src/components/ToggleView/ToggleView.tsx

```diff
@@ -0,0 +1,48 @@
+import { ViewState } from '../../types/view';
+import './Toggle.scss';
+
+const iconPaths: Record<ViewState, string> = {
+    grid: '/src/assets/Icons-Grid.svg',
+    list: '/src/assets/Icons-List.svg'
+};
+
+function ToggleButton({ currentView, onToggle }: ToggleViewProps) {
```

Comment:
optional: use `props: ToggleViewProps` for defining props, for the reason we discussed face to face.

In general an `optional` comment can be discarded, put please read it.